events/index.html

```diff
@@ -13,28 +13,37 @@
     Events description...
 {%- endblock %}
 
-{% block hero %}
-    {# Only show the hero when:
-       1. We are on the first page of paginated results
-       2. A hero exists
-    #}
 
-    {%
-        set current_page = 1
-    %}
+{# Only show the hero when:
+    1. We are on the first page of paginated results
+    2. A hero exists
+#}
 
-    {%
-        set hero = true
-    %}
+{# TODO remove commented line #}
+{# set show_hero = (vars.events.current_page == 1 and view.hero) %#}
+{% set show_hero = true %}
 
-    {#{% if vars.events.current_page == 1 and view.hero %}#}
-    {% from "_event-landing-macros.html" import event_hero as event_hero %}
-    {% if current_page == 1 and hero %}
-        {{ event_hero(hero) }}
+{% block content_sidebar_modifiers %}
+    {{ super() }}
+    {% if show_hero %}
+        u-hide-on-mobile
     {% endif %}
+{% endblock %}
 
+{% block hero %}
+    {% if show_hero %}
+        {% import "sidenav.html" as sidenav %}
+        {{ sidenav.render(
```

Comment:
There are duplicated `nav-secondary` blocks because it's also pulled in by the sidebar.